### ISSUE TITLE
Percentage value are calculated by row instead of by column

### ISSUE THREAD

**Issue by totocaca123** [jenkinsci/nested-data-reporting-plugin#47]
### Jenkins and plugins versions report

<details>
  <summary>Environment</summary>

  
  ```text
Jenkins: 2.361.1
OS: Windows 11 - 10.0
---
ace-editor:1.1
analysis-model-api:10.16.1
ant:475.vf34069fef73c
antisamy-markup-formatter:2.7
apache-httpcomponents-client-4-api:4.5.13-138.v4e7d9a_7b_a_e61
bootstrap4-api:4.6.0-5
bootstrap5-api:5.2.0-3
bouncycastle-api:2.26
branch-api:2.1046.v0ca_37783ecc5
build-failure-analyzer:2.4.0
build-monitor-plugin:1.13+build.202205140447
build-timeout:1.24
caffeine-api:2.9.3-65.v6a_47d0f4d1fe
checks-api:1.7.5
cloudbees-folder:6.758.vfd75d09eea_a_1
command-launcher:84.v4a_97f2027398
content-replace:1.7.0
credentials:1143.vb_e8b_b_ceee347
credentials-binding:523.vd859a_4b_122e6
data-tables-api:1.12.1-3
display-url-api:2.3.6
durable-task:500.v8927d9fd99d8
echarts-api:5.3.3-1
email-ext:2.91
font-awesome-api:6.1.2-1
forensics-api:1.15.1
git:4.11.5
git-client:3.11.2
git-parameter:0.9.17
git-server:99.va_0826a_b_cdfa_d
github:1.35.0
github-api:1.303-400.v35c2d8258028
github-branch-source:1694.vd46793a_c4a_57
gradle:1.39.4
handlebars:3.0.8
instance-identity:116.vf8f487400980
jackson2-api:2.13.3-285.vc03c0256d517
jakarta-activation-api:2.0.1-1
jakarta-mail-api:2.0.1-1
javax-activation-api:1.2.0-4
javax-mail-api:1.6.2-7
jaxb:2.3.6-1
jdk-tool:55.v1b_32b_6ca_f9ca
jjwt-api:0.11.5-77.v646c772fddb_0
jnr-posix-api:3.1.15-1
jquery:1.12.4-1
jquery3-api:3.6.0-4
jsch:0.1.55.61.va_e9ee26616e7
junit:1119.1121.vc43d0fc45561
ldap:2.12
lockable-resources:2.16
mailer:438.v02c7f0a_12fa_4
matrix-auth:3.1.5
matrix-project:785.v06b_7f47b_c631
memory-map:2.2.1 (53614)
metrics:4.2.10-389.v93143621b_050
mina-sshd-api-common:2.8.0-36.v8e25ce90d4b_1
mina-sshd-api-core:2.8.0-36.v8e25ce90d4b_1
momentjs:1.1.1
nested-data-reporting:4.0.0
okhttp-api:4.9.3-108.v0feda04578cf
pam-auth:1.10
pipeline-build-step:2.18
pipeline-github-lib:38.v445716ea_edda_
pipeline-graph-analysis:195.v5812d95a_a_2f9
pipeline-groovy-lib:612.v84da_9c54906d
pipeline-input-step:449.v77f0e8b_845c4
pipeline-milestone-step:101.vd572fef9d926
pipeline-model-api:2.2114.v2654ca_721309
pipeline-model-definition:2.2114.v2654ca_721309
pipeline-model-extensions:2.2114.v2654ca_721309
pipeline-rest-api:2.24
pipeline-stage-step:296.v5f6908f017a_5
pipeline-stage-tags-metadata:2.2114.v2654ca_721309
pipeline-stage-view:2.24
pipeline-utility-steps:2.13.0
plain-credentials:139.ved2b_9cf7587b
plugin-util-api:2.17.0
popper-api:1.16.1-3
popper2-api:2.11.6-1
prism-api:1.28.0-2
resource-disposer:0.20
scm-api:621.vda_a_b_055e58f7
script-security:1175.v4b_d517d6db_f0
snakeyaml-api:1.31-84.ve43da_fb_49d0b
ssh-credentials:305.v8f4381501156
ssh-slaves:1.834.v622da_57f702c
sshd:3.249.v2dc2ea_416e33
structs:324.va_f5d6774f3a_d
timestamper:1.19
token-macro:308.v4f2b_ed62b_b_16
trilead-api:2.72.v2a_3236754f73
variant:59.vf075fe829ccb
warnings-ng:9.19.1
workflow-aggregator:590.v6a_d052e5a_a_b_5
workflow-api:1192.v2d0deb_19d212
workflow-basic-steps:994.vd57e3ca_46d24
workflow-cps:2759.v87459c4eea_ca_
workflow-durable-task-step:1199.v02b_9244f8064
workflow-job:1232.v5a_4c994312f1
workflow-multibranch:716.vc692a_e52371b_
workflow-scm-step:400.v6b_89a_1317c9a_
workflow-step-api:639.v6eca_cd8c04a_a_
workflow-support:838.va_3a_087b_4055b
ws-cleanup:0.43
  ```

</details>

### What Operating System are you using (both controller, and any agents involved in the problem)?

windows 11

### Reproduction steps

build report and view distribution area

### Expected Results

I expect to see percentage relative to Total numbers.

### Actual Results

Percentage are calculated from row total (similar to "Distribution" column)
I don't know if it is a bug or if it could be an option.

![image](https://user-images.githubusercontent.com/42418580/188957982-eceafc3d-4822-4d3e-9ad4-63dbe92562b7.png)

**Comment by simonsymhoven** [comment]
@totocaca123 DO you think like this:

![Bildschirmfoto 2022-09-08 um 13 58 32](https://user-images.githubusercontent.com/26878018/189116365-0fef5473-945b-43fd-bbb5-f88028c1500e.png)

now I divide each value by the total of the actual view, e.g.:

Aktie (incorrect) 13902/(44907+2832+41942) = 15,41%

So the vertical sum is correct now for absolute and relative values.

Or we choose the second approach, an divide the value only by the corresponding group, e.g.

Aktie (incorrect) 13902/(44907) = 30,96%, so the vertical sum will result in 100% for the relative values. 

What do you think?

**Comment by dragoudin** [comment]
For 1 column data, the result is the same.
For multi column, I expect to have 100% for sum of all incorrect (usually, I want to know percent of total incorrect).

For the total line, we don't need percent (maybe to have the distribution column available for the total row).
To know the proportion of each column, we can see it in the overview pie chart. Maybe an option to view percent in "overview" area would be usefull in legend

In this description to have percent  (to see if it's usefull) but from the pie chart, i have an idea of percent of each column
![image](https://user-images.githubusercontent.com/94975153/189165473-c576447e-0a28-4a42-8622-b301168e9546.png)

**Comment by simonsymhoven** [comment]
that definitely makes sense. the distribution column in the total line is useless, because that exactly the pie chart. As well as the relative values in the total line are exactly the values form the pie chart. 

![Bildschirmfoto 2022-09-08 um 19 47 46](https://user-images.githubusercontent.com/26878018/189191155-ac0705ec-e19e-4da9-8121-8bec4ecf2684.png)

**Comment by simonsymhoven** [comment]
fixed with #51
